merlin/core/dispatch.py

```diff
@@ -414,8 +414,10 @@ def parquet_writer_dispatch(df: DataFrameLike, path=None, **kwargs):
             _args.append(pa.Table.from_pandas(df, preserve_index=False).schema)
     elif cudf is not None:
         _cls = cudf.io.parquet.ParquetWriter
-    else:
-        ValueError("Unable to load cudf. Please check your environment GPU and cudf available.")
+    elif cudf is None:
+        ValueError(
```

Comment:
I think this should raise?

Reply:
🤦 I was looking at this and wondering why pylint was confused about not reaching this line. And somehow I was blind to the missing raise here!